gloo/docs/upgrading/0.18.0.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Upgrading to 0.18.0
+weight: 1
+---
+
+# Upgrading to 0.18.0 on Kubernetes
+In Gloo 0.18.0, we updated the Gateway API to support managing Envoy TCP (L4) routing configuration in addition to existing HTTP/S (L7). This required making a non-backwards compatible change to the Gateway CRD in Gloo. 
+
+Due to limited support for CRD versioning in older versions of Kubernetes (we cannot assume our customers run Kubernetes 1.13+), we implemented this change with a new Gateway v2 CRD. Gloo 0.18.0+ will no longer install the old Gateway CRD or controller.  
+
+We recommend that you update your load balancer to handle routing traffic to two services: `gateway-proxy` and the new `gateway-proxy-v2`. If that is not possible in your environment, you can follow the steps below. However, installation for subsequent versions of Gloo may assume that you are running `gateway-proxy-v2` exclusively.
+
+## Upgrade Steps
+
+This guide documents a process for safely upgrading Gloo to 0.18.0, with an emphasis on minimizing manual work and avoiding downtime while shifting traffic to the new Gateway implementation. 
+
+### 1. Prepare your existing resources for upgrade
+To facilitate future updates, we've introduced live labels to our gateway-proxy deployments and services. You can take advantage of this feature during the migration to v0.18.0 by updating your existing resources. Run `kubectl edit deployment -n gloo-system gateway-proxy` and add `gateway-proxy: live` to the labels map under spec.template.metadata. Similarly, run `kubectl edit service -n gloo-system gateway-proxy`, add `gateway-proxy: live` to the selector map under spec, and remove all other selectors. Your gateway should function as before.
```

Comment:
I think it might be worth making it more clear that without the new label the transition will not work seamlessly

Reply:
done added a line about that